validator/keymanager/v2/derived/derived.go

```diff
@@ -258,6 +258,20 @@ func (dr *Keymanager) ValidatingAccountNames(ctx context.Context) ([]string, err
 	return names, nil
 }
 
+// ValidatingPublicKeys for the derived keymanager.
+func (dr *Keymanager) ValidatingPublicKeys() ([][48]byte, error) {
```

Comment:
Is this related to the fix? I have a feeling we might need to split this into two prs

Reply:
Yes, this is part of the fix. The only thing that is not directly a fix of this problem but is related is this:
https://github.com/prysmaticlabs/prysm/pull/6978/commits/18ff2fb6522854c6c126622b6e124cee59572162

Reply:
Thanks for confirming. I think that's ok to include. 
Perhaps add a note in the description and mention there's 2 fixes and the intention of each fix